error-handler.js

```diff
@@ -269,6 +277,9 @@ createHandler = function createHandler(options) {
       return resumeOrClose(status);
     }
 
+    // Always set a response status.
```

Comment:
I see.. maybe we should only modify the response object in a single location (`send()`) and do this modification on the line right before `res.send()`.